addons/payment_paypal/models/payment_transaction.py

```diff
@@ -10,7 +10,6 @@
 from odoo.addons.payment_paypal import utils as paypal_utils
 from odoo.addons.payment_paypal.const import PAYMENT_STATUS_MAPPING
 
-
```

Comment:
Following PEP 8's recommendations, there should be two blank lines between the imports and `_logger`.

Reply:
Same for other files changed in this PR.